clover_web.rb

```diff
@@ -328,6 +328,17 @@ def assets
     auto_add_recovery_codes? true
     auto_remove_recovery_codes? true
     recovery_auth_view { view "auth/recovery_auth", "Recovery Codes" }
+
+    # Password Requirements
+    password_minimum_length 8
+    password_maximum_bytes 72
+    password_meets_requirements? do |password|
+      return password.match?(/[a-z]/) && password.match?(/[A-Z]/) && password.match?(/[0-9]/)
```

Comment:
do you really mean `return` here?

Reply:
you are right, it is not necessary. removed it.